diff -up CherryPy-2.3.0/cherrypy/test/test_core.py.py26 CherryPy-2.3.0/cherrypy/test/test_core.py
--- CherryPy-2.3.0/cherrypy/test/test_core.py.py26	2008-12-01 11:28:19.000000000 -0800
+++ CherryPy-2.3.0/cherrypy/test/test_core.py	2008-12-01 11:28:36.000000000 -0800
@@ -52,7 +52,7 @@ def setup_server():
         and adds an instance of the subclass as an attribute of cherrypy.root.
         """
         def __init__(cls, name, bases, dct):
-            type.__init__(name, bases, dct)
+            type.__init__(cls, name, bases, dct)
             for value in dct.itervalues():
                 if isinstance(value, types.FunctionType):
                     value.exposed = True
diff -up CherryPy-2.3.0/cherrypy/test/test_custom_filters.py.py26 CherryPy-2.3.0/cherrypy/test/test_custom_filters.py
--- CherryPy-2.3.0/cherrypy/test/test_custom_filters.py.py26	2008-12-01 11:28:29.000000000 -0800
+++ CherryPy-2.3.0/cherrypy/test/test_custom_filters.py	2008-12-01 11:28:49.000000000 -0800
@@ -79,7 +79,7 @@ def setup_server():
         and adds an instance of the subclass as an attribute of cherrypy.root.
         """
         def __init__(cls, name, bases, dct):
-            type.__init__(name, bases, dct)
+            type.__init__(cls, name, bases, dct)
             for value in dct.itervalues():
                 if isinstance(value, types.FunctionType):
                     value.exposed = True
